CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # clx 0.18.0 (Date TBD)
 
 ## New Features
+- PR #328 SlashNext API integration
```

Comment:
just a reminder that manual changelog entries are no longer necessary. There's no harm in leaving this. It will be overwritten during the next release.

Reply:
@ajschmidt8 thanks for the reminder. I will leave it for now.